vendor/engines/secure_rooms/app/controllers/secure_rooms/card_readers_controller.rb

```diff
@@ -0,0 +1,81 @@
+module SecureRooms
+
+  class CardReadersController < ApplicationController
+
+    admin_tab :all
+
+    layout "two_column"
+
+    before_action :authenticate_user!
+    before_action :check_acting_as
+    before_action :authorize_card_reader
+    before_action :init_current_facility
+    before_action :init_product
+    before_action :load_card_reader, except: [:index]
+
+    def initialize
+      @active_tab = "secure_rooms"
+      super
+    end
+
+    def index
```

Comment:
Here's a question, and I'm not 100% sure the answer to: should facility staff still be able to see the index page?

If so, then we need to hide their ability to see the edit/remove links on the index page.

If we decide they don't need to see the index view, then we should probably hide the tab, otherwise it will 403.